recipes/hoomd-dlext/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set version = "0.2.1" %}
+{% set build = 0 %}
+
+package:
+  name: hoomd-dlext
+  version: {{ version }}
+
+source:
+  url: https://github.com/SSAGESLabs/hoomd-dlext/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 547316c95608d7ee1ca11ace53ccf22dddc88206fa0ac0613020f48bf6081269
+
+build:
+  number: {{ build }}
```

Otherwise the bot might not be able to automatically bump the build no on PRs
```suggestion
  number: 0
```